src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog5_raw.xml.sarif

```diff
@@ -47,6 +47,30 @@
           }
         ]
       },
+      "logicalLocations": {
+        "AndroidStudioFormatExample": [
+          {
+            "name": "AndroidStudioFormatExample",
+            "kind": "android-module"
+          }
+        ],
+        "app": [
+          {
+            "name": "app",
```

Comment:
is the name optional in cases where it matches the key or do we always require it?